packages/bpk-component-link/src/BpkButtonLink.tsx

```diff
@@ -29,13 +26,14 @@ import STYLES from './BpkLink.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-type Props = {
-  children: Node,
-  className: ?string,
-  onClick: (event: SyntheticEvent<>) => mixed,
-  alternate: boolean,
+export interface Props extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'className' | 'onClick'> {
+  /** The content of the link. */
```

Comment:
After adding JSDoc comments, `react-docgen-typescript` will recognize children as props with custom documentation, and they should appear in the API documentation